src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -219,8 +219,11 @@ const constructPageFetchUrl = ({
         fetchUrl = Url(`/${id}`);
         break;
       case HOME_PAGE: {
-        const variantPath = variant ? `/${variant}` : '';
-        fetchUrl = Url(`/${service}${variantPath}`);
+        const host = `http://${process.env.HOSTNAME || 'localhost'}`;
+        const port = process.env.PORT ? `:${process.env.PORT}` : '';
+        fetchUrl = Url(
+          `${host}${port}/api/local/${service}/homePage/${variant ? `${variant}` : 'index'}`,
+        );
```

Comment:
We need to be able to support both ways, because homepages are still rendered by the Express app. 

We need to check if we're on next JS (is there an environment variable we can use?) and do this, otherwise, use the previous logic (of `/service/variant`)